website/src/parsers/js/typescript-eslint-parser.js

```diff
@@ -37,6 +39,8 @@ export default {
   _getSettingsConfiguration(defaultOptions) {
     return {
       fields: [
+        ['ecmaVersion', [3, 5, 6, 7, 8, 9], value => Number(value)],
```

Comment:
Can’t this just be `Number`?

```suggestion
        ['ecmaVersion', [3, 5, 6, 7, 8, 9], Number],
```

Reply:
Probably, I replicated the options from https://github.com/fkling/astexplorer/blob/master/website/src/parsers/js/espree.js#L54 for a bit of consistency.